src/components/Regiser.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Logo from './Logo.js';
+import {Link, Redirect} from 'react-router-dom';
+
+export default class Regiser extends Component {
+    render() {
+        return (
+            <div>
+                <div className="logoposi"><Logo variant="big"/></div>
+                <div className="row">
+                    <div className="col-2"></div>
+                        <div className="col-8">
+                            <input id="registerinputs" onChange={this.updateMail} className="logininInput" type="text" placeholder="Full name"/></div>
+                        <div className="col-2"></div>
+                        
+                        <div className="col-2"></div>
+                        <div id="registerinputs" className="col-8">
+                            <input onChange={this.updateMail} className="logininInput" type="text" placeholder="Your Email"/></div>
+                        <div className="col-2"></div>
+
+                        <div className="col-2"></div>
+                        <div className="col-8">
+                            <input id="registerinputs" onChange={this.updateMail} className="logininInput" type="text" placeholder="Password"/></div>
+                        <div className="col-2"></div>
+
+                        <div className="col-2"></div>
+                        <div className="col-8">
+                            <input id="registerinputs" onChange={this.updateMail} className="logininInput" type="text" placeholder="Repeat password"/></div>
```

Comment:
All the inputs has the same id, "registerinputs".
Id should come only once in single element.
If you need to use the same css on many elements change it to class